### PR TITLE
fix: default and named exports

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: .babelrc.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 module.exports = api => ({
   presets: [
+    '@babel/preset-typescript',
```
Comment (itajaja) [inline .babelrc.js]: ooh so maybe that's why it was using import `syntax`! are yo fixing upstream?

Comment (jquense) [reply .babelrc.js]: no :/ this didn't have an effect

**Comment by jquense** [comment]
> wait, what's the actual difference in the output?

The current output is just `exports.default = Env` which means you need to do: `require('@4c/env').default`. this change allows `require('@4c/env')`

**Comment by itajaja** [comment]
I can't believe there is no way to fix this otherwise 😱

**Comment by jquense** [comment]
I mean this behavior i'm insisting on is really less correct, it's just what we are used too.

File: src/env.ts [hunk 2]
```diff
@@ -13,7 +13,8 @@ function loadFromFile(env = 'test') {
   return parsed!; // if error is undefined, then parse is defined
 }
 
-export default {
+// add-modules-exports doesn't work with ts!?
+exports.default = {
```
Comment (taion) [inline src/env.ts]: is there a point to doing this if we're only consuming the CJS transpiled version?

Comment (jquense) [reply src/env.ts]: probably not!

**Pull request merged**